mmv1/provider/terraform.go

```diff
@@ -144,6 +144,28 @@ func (t *Terraform) GenerateResource(object api.Resource, templateData TemplateD
 	}
 }
 
+func (t *Terraform) GenerateResourceTests(object api.Resource, templateData TemplateData, outputFolder string) {
+	eligibleExample := false
+	for _, example := range object.Examples {
+		if !example.SkipTest {
+			if object.ProductMetadata.VersionObjOrClosest(t.Version.Name).CompareTo(object.ProductMetadata.VersionObjOrClosest(example.MinVersion)) > 0 {
+				eligibleExample = true
```

Comment:
Should jump out the loop with `break` statement once `eligibleExample` is `true`?

Reply:
yes you are right! Added